### DISCUSSION TITLE
Need help getting errors

### DISCUSSION THREAD

**Discussion by AmibeSkyfy16** [vincentlauvlwj/Ktorm#423] [Show and tell]
Hello, I have an error that I can't solve. this error -> https://pastebin.com/budFvPiR

I understand what she means, but not why it happens. I know I'm doing something wrong in the code, but I don't know what

Here is some image of my code (whole code [here](https://github.com/AmibeSkyfy16/TinyEconomyRenewed/blob/master/src/main/kotlin/ch/skyfy/tinyeconomyrenewed/TinyEconomyRenewedInitializer.kt))

![image](https://user-images.githubusercontent.com/85696108/185758373-6c45a47d-bd14-460d-a8f0-5a6d92593d0c.png)
![image](https://user-images.githubusercontent.com/85696108/185758391-d1338059-9e55-4e68-afdf-9fc42529f7ed.png)

**Comment by vincentlauvlwj** [comment]
But this problem is not about Ktorm.

**Reply by AmibeSkyfy16** [reply]
Yes, sorry, I thought that "show and tell" was the right channel. A channel where you can talk about one of these projects that uses ktorm and at the same time errors can be linked to it indirectly. But anyway, I think I solved the problem. Have a nice day
